config/manifests/bases/oadp-operator.clusterserviceversion.yaml

```diff
@@ -314,6 +314,164 @@ spec:
         displayName: Phase
         path: phase
       version: v1
+    - description: DataDownload represents a data download of a volume snapshot. There
```

Comment:
Lets wait for rebase to get @shubham-pampattiwar PR for descriptions of DU and DD CRDs?

Reply:
yeah @mateusoliveira43 I can do a cherry-pick against openshift/velero if needed, [upstream PR](https://github.com/vmware-tanzu/velero/pull/7028) is merged @mpryc

Reply:
We won't be doing any more rebases to konveyor-dev until we've created oadp-1.3 branches, since we're already on the upstream release we're shipping for 1.3 -- any upstream commits needed for 1.3 will need to be cherry-picked via PR.

Reply:
ok ok, I will create  a cherry-pick PR

Reply:
So there won't be any rebase meaning this PR is ok'ish and should be valid for the master. If there is any update removal of description needed following this patch it should be removed with PR. Am I understanding this correctly?

Reply:
I would remove changes related to DUCR and DDCR from this PR. As they will be automatically picked up once the Velero CRDs are updated.

Reply:
Will they? This is the bases manifest for the top level CSV so I don't think we have any automagic which is updating this file?

Reply:
> As they will be automatically picked up once the Velero CRDs are updated.

The description wouldn't be picked up automatically.

The operator-sdk bundle generation only automate Name, Version, Kind into bundle/CSV.

```
	for _, defKey := range defKeys {
		if owned, ownedExists := descMap[defKey]; ownedExists {
			ownedDescs = append(ownedDescs, owned)
		} else {
			ownedDescs = append(ownedDescs, operatorsv1alpha1.CRDDescription{
				Name:    defKey.Name,
				Version: defKey.Version,
				Kind:    defKey.Kind,
			})
		}
	}
```

https://redhat-internal.slack.com/archives/C0144ECKUJ0/p1698773258374849

Still have to put in description manually and phase/specDescriptors if desired.